src/content/docs/tutorial-app-performance/guide-to-monitoring-core-web-vitals.mdx

```diff
@@ -19,11 +19,34 @@ import browserPageUrlView from 'images/browser_screenshot-full_page-url-view.web
 
 import browserFilteredWebVitals from 'images/browser_screenshot-full_filtered-web-vitals.webp'
 
-Core web vitals are Google’s metrics to gauge overall user experience of your site, which can influence your site’s SEO rankings and give you valuable insight into how users perceive your business.  
+Core web vitals are Google’s metrics that gauge overall user experience. They give valuable insight into how users perceive your business, which can influence your site’s SEO rankings. By having a score that captures user experience, you can take action on specific problems rather than guessing at what your site needs.
```

Comment:
Perceive reads a little strange here. Might not be worth fixing but thought I'd bring it up.